src/instructlab/model/evaluate.py

```diff
@@ -125,9 +129,53 @@ def evaluator_run(evaluator, api_base):
             few_shots,
             batch_size,
             tasks_dir,
+            input_questions,
         )
 
-        if benchmark == Benchmark.MT_BENCH:
+        if benchmark == Benchmark.DK_BENCH:
```

Comment:
one last question here, it seems this Try Except is meant to catch `EvalError`, is that the only exception type raised from the new `run_dk_bench`?